packages/inter-protocol/src/auction/scheduler.js

```diff
@@ -205,11 +213,12 @@ export const makeScheduler = async (
     publishSchedule();
   };
 
+  // XXX liveSchedule and nextSchedule are in scope
```

Comment:
It's tech debt to me that the function takes a parameter that it already has. Giving it the parameter was done for clarity. I think it needs further refactoring. I'll do it in a fixup

Reply:
I'm ambivalent. Not relying on it being in scope makes a potential future refactor easier. I probably shouldn't mention _nits_ at this point because they're not worth fixing.